WordPress/Classes/System/WordPressAppDelegate.m

```diff
@@ -178,10 +178,7 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
 {
     DDLogVerbose(@"didFinishLaunchingWithOptions state: %d", application.applicationState);
 
-    // Launched by tapping a notification
-    if (application.applicationState == UIApplicationStateActive) {
```

Comment:
Oh, yah, that comment could be better for sure. 
The actual reason to keep that check would be to avoid processing remote notifications received while in a background application state.